### PR TITLE
Fix. Contributing instructions

### PR DESCRIPTION
**Pull request by LiraLemur**
### Description of the Change

Updated instructions for contributors after we switched to JIRA from GitHub issues

### Benefits

A little more clarity

### Possible Drawbacks 

See none. If it's ok and no ideas come up, I'll also put that into the documentation later. 

### Alternate Designs *[optional]*

We could simply move it all to documentation. What do you think?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,46 +38,40 @@ The following is a short set of guidelines for contributing to Iroha.
...
 
 
-### Suggesting Enhancements
+### Suggesting Improvements
 
-An *enhancement* is a code or idea, which makes **existing** code or design faster, more stable, portable, secure or better in any other way.
+Improvement is a code or idea, which makes **existing** code or design faster, more stable, portable, secure or better in any other way.
 
-Enhancements are tracked as [GitHub Issues](https://guides.github.com/features/issues/). To submit new enhancement, create new Issue and incllude these details:
+Improvements are also tracked in [Hyperledger Jira](https://jira.hyperledger.org). To submit new improvement idea, create a new [Issue](https://jira.hyperledger.org/secure/CreateIssue!default.jspa) and include these details:
```
Comment (neewy) [inline CONTRIBUTING.md]: Aren't we using Ideas portal here? https://iroha.ideas.aha.io

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,145 +2,108 @@
...
 - [Asking Questions](#asking-questions)
 - [Your First Code Contribution](#your-first-code-contribution)
 - [Pull Requests](#pull-requests)
 
 ##### [Styleguides](#styleguides-1)
 
-- [Git Commit Messages](#git-commit-messages)
-- [C++ StyleGuide](#C++-styleguide)
+- [Git Style Guide](#git-style-guide)
+- [C++ Style Guide](#c++-style-guide)
```
Comment (lebdron) [inline CONTRIBUTING.md]:
```suggestion
- [C++ Style Guide](#c-style-guide)
```
image

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,145 +2,108 @@
...
+- **Write tests** for new code. Test coverage for new code must be at least 70%
 
-- Do not include issue numbers in the PR title or commit messages.
+- Every pull request should be reviewed and **get at least two approvals from maintainers team**. Check who is a current maintainer in [MAINTAINERS.md](https://github.com/hyperledger/iroha/blob/master/MAINTAINERS.md) file
 
-- Use [keywords for closing issues](https://help.github.com/articles/closing-issues-using-keywords/).
+- When you've finished work make sure that you've got all passing CI checks — after that **squash and merge** your pull request
 
-- Include issue numbers in Pull Request body only.
+- Follow the [C++ Style Guide](#c++-style-guide)
```
Comment (lebdron) [inline CONTRIBUTING.md]: Same as above.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,145 +2,108 @@
...
+A *question* is any discussion that is typically neigher a bug, nor feature request or improvement. If you have a question like "How do I do X?" - this paragraph is for you.
 
+Please post your question in [your favourite messenger](#places-where-community-is-active) so members of the community could help you. You can also help others!
 
 ### Your First Code Contribution
 
-Read our [C++ Style Guide](#c++-style-guide) and start with beginner-friendly issues with label [`[good-first-issue]`](https://github.com/hyperledger/iroha/issues?q=is:open+is:issue+label:good-first-issue ). Indicate somehow that you are working on this task.
-
-
+Read our [C++ Style Guide](#c++-style-guide) and start with beginner-friendly issues with JIRA label [`[good-first-issue]`](https://jira.hyperledger.org/issues/?jql=project%20%3D%20IR%20and%20labels%20%3D%20good-first-issue%20ORDER%20BY%20updated%20DESC) <- click here. Indicate somehow that you are working on this task: get in touch with maintainers team, community or simply assign this issue to yourself.
```
Comment (lebdron) [inline CONTRIBUTING.md]: Same as above.

**Pull request merged**